hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -542,30 +584,38 @@ static QString* psh_convert(struct PixelShader *ps)
     /* Window Clipping */
     QString *clip = qstring_new();
     if (ps->state.window_clip_count != 0) {
-        qstring_append_fmt(preflight, "uniform ivec4 clipRegion[%d];\n",
+        qstring_append_fmt(preflight,
+                           "uniform ivec4 clipRegion[%d];\n",
                            ps->state.window_clip_count);
-        qstring_append_fmt(clip, "/*  Window-clip (%s) */\n",
-                           ps->state.window_clip_exclusive ?
-                               "Exclusive" : "Inclusive");
+        qstring_append_fmt(clip,
+                           "/*  Window-clip (%s) */\n",
+                           ps->state.window_clip_exclusive ? "Exclusive" :
+                                                             "Inclusive");
         if (!ps->state.window_clip_exclusive) {
             qstring_append(clip, "bool clipContained = false;\n");
         }
-        qstring_append_fmt(clip, "for (int i = 0; i < %d; i++) {\n",
+        qstring_append_fmt(clip,
+                           "for (int i = 0; i < %d; i++) {\n",
                            ps->state.window_clip_count);
-        qstring_append(clip, "  bvec4 clipTest = bvec4(lessThan(gl_FragCoord.xy, clipRegion[i].xy),\n"
-                             "                         greaterThan(gl_FragCoord.xy, clipRegion[i].zw));\n"
-                             "  if (!any(clipTest)) {\n");
+        qstring_append(clip,
+                       "  bvec4 clipTest = bvec4(lessThan(gl_FragCoord.xy, "
+                       "clipRegion[i].xy),\n"
```

Comment:
Yuck! Cutting GLSL into multiline here is a bad idea in my opinion.
I think it's a good case where ignoring the 80 col limit is much more beneficial.